docs/notebooks/markov-models.ipynb

```diff
@@ -137,7 +137,7 @@
     "The symbol etymology, which usually gets swept under the rug in mathematically-oriented papers, are as follows: \n",
```

Comment:
<p>"the mean gives us information <strong>about the state</strong>, but the scale doesn't, and at the same time, the mean depends only on the <strong>current</strong> state, and not on the previous state."</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/ericmjl/bayesian-analysis-recipes/pull/10/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='47'/>

Reply:
<p>I've rephrased it after another colleague also caught the phrasing being off.</p>